.github/workflows/tf-account.yaml

```diff
@@ -12,12 +12,25 @@ on:
         required: true
       TF_BACKEND_S3_REGION:
         required: true
+      TS_OAUTH_CLIENT_ID:
+        required: false
+      TS_OAUTH_SECRET:
+        required: false
     inputs:
       terraform-version:
         description: 'Terraform version'
         required: false
         type: string
         default: '1.5.4'
+      TS_TAGS:
+        required: false
+        type: string
+      HTTPS_PROXY:
```

Comment:
Is this not related to Tailscale? I thought that the idea is essentially that the Tailscale connection is essentially providing the proxy.

Reply:
tailscale connects you to a network, you still have to provide the egress address.